sn_protocol/src/messages/mod.rs

```diff
@@ -28,7 +28,7 @@ use crate::{
 };
 use serde::{Deserialize, Serialize};
 use sn_dbc::SignedSpend;
-use sn_registers::Register;
+use sn_registers::SignedRegister;
```

Comment:
do we have the user scenario to store/operate un-signed register on network?
If don't, maybe just `assume register shall always be signed` ?
Then we can just stay with the existing name of `Register`, but they shall always be signed?

I just feel the current `Register` and `SignedRegister` are bit confusing and misleading

Reply:
`Register` is the local copy of your register, it's the one you're operating on, `SignedRegister` is the signed/verifiable version that we store on the Network. In theory, users should only deal with `Registers` and Network only with `SignedRegisters`.